plugins/github-actions/src/components/Router.tsx

```diff
@@ -23,30 +23,39 @@ import {
 import { Routes, Route } from 'react-router-dom';
 import { buildRouteRef } from '../routes';
 import { WorkflowRunDetails } from './WorkflowRunDetails';
+import { WorkflowRunsCard } from './WorkflowRunsCard';
 import { WorkflowRunsTable } from './WorkflowRunsTable';
 import { GITHUB_ACTIONS_ANNOTATION } from './getProjectNameFromEntity';
+import { RouterProps, ViewEnum } from '../api/types';
 
 /** @public */
 export const isGithubActionsAvailable = (entity: Entity) =>
   Boolean(entity.metadata.annotations?.[GITHUB_ACTIONS_ANNOTATION]);
 
 /** @public */
-export const Router = () => {
+export const Router = ({ view = ViewEnum.Table }: RouterProps) => {
   const { entity } = useEntity();
```

Comment:
```suggestion
export const Router = (props: RouterProps) => {
  const { view = ViewEnum.Table } = props;
  const { entity } = useEntity();
```

Avoid destructuring in the parameter list for public APIs, since that pollutes the declaration in docs

Reply:
Fixed